lib/routes/data/styles.js

```diff
@@ -63,6 +63,20 @@ module.exports = {
       ],
     ],
   },
+  contentData: {
+    name: 'Careers',
+    description: 'A section realted to joining Watson and what the culture of Watson is about',
```

Comment:
Just for clarity sake, can you make this description sound more like it's describing a content type instead of a section of a website? Slight wording change should prol be enough